packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -269,8 +269,7 @@
         min-width: 15rem;
         width: 100%;
 
-        &:not(.cfd-personal-details-modal__form *):not(.trade-container__multiplier-dropdown):not(
-                .dc-dropdown--left):not(.contract-type-info__dropdown) {
+        &:not(.cfd-personal-details-modal__form *):not(.trade-container__multiplier-dropdown):not(.dc-dropdown--left):not(.contract-type-info__dropdown) {
```

Comment:
using `:not` might cause issues with Safari browser, just like `:has` did for jurisdiction modal before

Reply:
This component's css change was done at the time of implementation of Jurisdictions and have not cause any issues since then. I don't think if this change is necessary.

Reply:
sure